app/src/main/scala/org/alephium/explorer/persistence/schema/BlockDepsSchema.scala

```diff
@@ -31,7 +31,6 @@ object BlockDepsSchema extends Schema[BlockDepEntity]("block_deps") {
     def depOrder: Rep[Int]         = column[Int]("dep_order")
 
     def pk: PrimaryKey = primaryKey("hash_deps_pk", (hash, dep))
-    def hashIdx: Index = index("deps_hash_idx", hash)
```

Comment:
I always thought that a tuple primary key was indexing the tuple as a whole, then we should be able to remove `deps_dep_idx` too no?

Reply:
Postgres doesn't seem to use the primary-key index if we query `dep` directly so we would need `deps_dep_idx` for those queries.

Reply:
SQL index is very counter-intuitive for me, it's hard to tell if they are used or not without using query planner.